tests/api_tests.js

```diff
@@ -322,8 +323,13 @@ function runNodeEnvTests(nodeEnvData, t) {
                                          t.ok(parseInt(nodeEnvData['max.old.space.size']) > 0,
                                               "max.old.space.size is positive");
 
-                                              var nodeVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
-                                              if(nodeVersion >= 6.5) { // issue 283
+                                              // converts e.g. 'v8.0.0' to [8,0.0]
```

Comment:
Fair enough, updated, PTAL.

Looking at it more, it's a pretty tiny dep anyway, so nbd to import.